src/components/Network.js

```diff
@@ -103,15 +125,9 @@ class Network extends React.Component {
     };
 
     const renderConnected = (peer, conn) => {
-        const sync = conn.plugins["node-sync-timestamp"];
-        const first_timestamp = this.state.dag.first_timestamp;
-        const latest_timestamp = this.state.dag.latest_timestamp;
-        const delta = latest_timestamp - first_timestamp;
-        const synced_percent = 100 * (sync.synced_timestamp - first_timestamp) / delta;
-        let general_percent = (100 * (sync.latest_timestamp - first_timestamp) / delta) - synced_percent;
-        if (general_percent > 100) {
-          general_percent = 100;
-        }
+        const sync_v1 = conn.plugins["node-sync-timestamp"];
+        const sync_v2 = conn.sync;
```

Comment:
This isn't quite right, `conn.sync` is actually the common denominator of all sync protocols and not exclusive to `sync_v2`. It might make sense that we have a `conn.sync.version`, but rather than that I think it'd be best if the code was actually less dependent on the sync version, the way it is now looks more dependent.

Maybe if we make an object that has all the sync data where fields from sync-v1 are OK to be null, like:

```
{
  "progress": 0.5,
  "state": "behind",
  "latest_timestamp": null,
  "synced_timestamp": null
}
```

The only way `conn.sync` is undefined is when the node is outdated, which I don't think we have to cover (updated explorer only works on updated node is a reasonable compromise).

Reply:
This new `sync data` seems a good idea, therefore I will make changes to expect this structure on the explorer.

Reply:
I followed this `sync data` structure, thanks for your suggestion.